tests_common/copy_files_from_core.py

```diff
@@ -43,16 +50,38 @@
 server_crt_file = os.path.join(data_dir, "server.crt")
 server_open_key_file = os.path.join(data_dir, "server.open.key")
 enduser_certs_dir = os.path.join(data_dir, "ca", "enduser-certs")
+simple_project_dir = os.path.join(data_dir, "simple_project")
+db_common_py = os.path.join(db_dir, "common.py")
+dest_db_common_py = os.path.join(dest_db_dir, "common.py")
+dump_tool_py = os.path.join(db_dir, "dump_tool.py")
+dest_dump_tool_py = os.path.join(dest_db_dir, "dump_tool.py")
+dest_simple_project_dir = os.path.join(dest_db_dir, "simple_project")
 
-if os.path.exists(dest_conftest_py):
-    os.remove(dest_conftest_py)
 
-if os.path.exists(dest_utils_py):
-    os.remove(dest_utils_py)
+def remove_file_if_exists(filename: str):
+    if os.path.exists(filename):
+        os.remove(filename)
+
+
+remove_file_if_exists(dest_conftest_py)
+remove_file_if_exists(dest_utils_py)
+remove_file_if_exists(dest_db_common_py)
+remove_file_if_exists(dest_dump_tool_py)
 
 # Copy files
 shutil.copy(conftest_py, dest_dir)
 shutil.copy(utils_py, dest_dir)
 shutil.copy(server_crt_file, dest_data_dir)
 shutil.copy(server_open_key_file, dest_data_dir)
 shutil.copytree(enduser_certs_dir, os.path.join(dest_data_dir, "ca", "enduser-certs"))
+shutil.copy(db_common_py, dest_db_common_py)
+shutil.copy(dump_tool_py, dest_dump_tool_py)
+shutil.copytree(simple_project_dir, dest_simple_project_dir)
+
+# Fix import
+with fileinput.input(dest_dump_tool_py, inplace=True) as f:
```

Comment:
please use a consistent solution with the conftest.py
it now has

```
# Import the utils module differently when conftest is put into the inmanta_tests package
if __file__ and os.path.dirname(__file__).split("/")[-1] == "inmanta_tests":
    from inmanta_tests import utils  # noqa: F401
else:
    import utils
```

(I'm undecided about which solution is worst)